toolkit/scripts/srpm_pack.mk

```diff
@@ -67,6 +67,7 @@ $(STATUS_FLAGS_DIR)/build_srpms.flag: $(local_specs) $(local_spec_dirs) $(local_
 		--tls-cert=$(TLS_CERT) \
 		--tls-key=$(TLS_KEY) \
 		--build-dir=$(BUILD_DIR)/SRPM_packaging \
-		--signature-handling=$(SRPM_FILE_SIGNATURE_HANDLING)
+		--signature-handling=$(SRPM_FILE_SIGNATURE_HANDLING) \
+		--log-file $(LOGS_DIR)/pkggen/workplan/intermediate_srpms.log --log-level $(LOG_LEVEL)
```

Comment:
nit: We don't do this everywhere, but please standardize the argument to match the above ones with `key=value`. So it would be `--log-file=....`.

Reply:
May as well fix this in the other places while I'm at it...

---
In reply to: [551631231](https://github.com/microsoft/CBL-Mariner/pull/480#discussion_r551631231) [](ancestors = 551631231)